packages/web-components/src/components/universal-banner/universal-banner.ts

```diff
@@ -76,17 +82,47 @@ class DDSUniversalBanner extends StableSelectorMixin(LitElement) {
     this.buttonCopy = hasContent[0]?.textContent?.trim();
   }
 
+  protected _handleResize() {
+    const { _breakpoint: breakpoint } = this;
+
+    this._shouldRenderAsLink = !breakpoint!.matches;
+    this.requestUpdate();
+  }
+
   firstUpdated() {
     StickyHeader.global.banner = this;
+
+    this._breakpoint = window.matchMedia(`(min-width: ${gridLgBreakpoint}px)`);
+    this._breakpoint.addEventListener('change', this._handleResize.bind(this));
+    this._handleResize();
   }
 
   updated() {
     this.querySelector((this.constructor as typeof DDSUniversalBanner).ctaButton)?.removeAttribute('role');
   }
 
-  render() {
+  /**
+   * Renders shadow dom within a static div
+   */
+  _renderAsStatic() {
     return html`
       <div class="${prefix}--universal-banner-layout-container">
+        ${this._renderInnerContents()}
+      </div>
+    `;
+  }
+
+  _renderAsLink() {
+    return html`
+      <a href="${this.buttonHref}" class="${prefix}--universal-banner-layout-container">
+        ${this._renderInnerContents()}
+      </a>
```

Comment:
@RichKummer @oliviaflory I'm just following up on the comment above and figured that I should include you both on this thread. Do you both know if we need to account for all CTA types for the CTA button used within the Universal Banner? For the Module 12A banner component being used on Drupal, we provided Content Authors the option to set the CTA as local, external, PDF/Document, video, etc (eg. https://pages.github.ibm.com/MSC-Cloud/hc_fractal/components/detail/12a--default.html). I figured that this type of option might be something that we want o replicate on the AEM side, but let me know if you guys have design guidelines defined for this functionality.

cc: @jwitkowski79 @kennylam

Reply:
@oliviaflory Makes sense! Gald that I asked because I've seen some instances in pages in Drupal where the CTA can be an `external` or `local` link, however, if this is not a convention that we're allowing for the `Universal Banner` component, I think that's fine, but we should note this in the content guideline and content entry process of this component content model within the AEM CMS, so that the Content Authors understand the limitation. @jwitkowski79 Can you take a look at this and let me know if this has already been documented somewhere? I believe we'll soon bring this component over to AEM as a way to promote site-wide announcements.

@andy-blum In the meantime, feel free to disregard my comment above as it sounds like we might just account for the `local` CTA icon type.

Reply:
@proeung Thanks for the heads up! We have not documented this component yet since it's not in AEM. However, I'll make a note of it so when we do bring it over to AEM, we can document the usage that Olivia mentioned above along with the proper CTA types

Reply:
Thanks, @jwitkowski79! I'll need to check in with Nathan, but I believe he mentioned wanting to bring this component into AEM soon.